test/dummy/config/environments/test.rb

```diff
@@ -29,6 +29,9 @@
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
 
+  # run tests in a sorted order (and get rid of rails its lengthy deprecation warning)
+  config.active_support.test_order = :sorted
```

Comment:
shouldn't the test order be random to catch potential order dependent test failures?